src/plugins/ovp/services/JwplayerMediaSubscriber.js

```diff
@@ -0,0 +1,80 @@
+import camelCase from 'lodash/camelCase';
+import EventSubscriber from '@gdbots/pbjx/EventSubscriber';
+import getTextFieldError from '@triniti/cms/components/text-field/getTextFieldError';
+
+export default class JwplayerMediaSubscriber extends EventSubscriber {
+  constructor() {
+    super();
+    this.onInitForm = this.onInitForm.bind(this);
+    this.onValidateForm = this.onValidateForm.bind(this);
+    this.onSubmitForm = this.onSubmitForm.bind(this);
+  }
+
+  /**
+   * Runs when the form is first created and is used to
+   * update initial values.
+   *
+   * @param {FormEvent} formEvent
+   */
+  onInitForm(formEvent) {
+    const data = formEvent.getData();
+    const node = formEvent.getMessage();
+
+    [
+      'jwplayer_media_id',
+      'jwplayer_sync_enabled',
+      'jwplayer_synced_at',
```

Comment:
@efjacobson `jwplayer_synced_at` is readonly, client cant' set it.  ref how the kaltura_synced_at works